src/Microsoft.MobileBlazorBindings.WebView.JS/src/RenderQueue.ts

```diff
@@ -0,0 +1,66 @@
+﻿import '@dotnet/jsinterop/dist/Microsoft.JSInterop';
+import { renderBatch } from '@browserjs/Rendering/Renderer';
+import { OutOfProcessRenderBatch } from '@browserjs/Rendering/RenderBatch/OutOfProcessRenderBatch';
+
+export class RenderQueue {
+    private static instance: RenderQueue;
+
+    private nextBatchId = 2;
```

Comment:
Why does this start at 2? Is that a magic number?

Reply:
Yeah, the first one that comes from C# is number one...

Reply:
I think this deserves a comment. The only numbers that don't require a comment are -MAX, -1, 0, 1, and MAX. This isn't one of those 😄